src/richie/apps/demo/defaults.py

```diff
@@ -16,7 +16,8 @@
     "licences": 5,
```

Comment:
Good point! The thing is that in real life our news are tagged with categories specific to news and there are less than 10 of them so my changes to the demo are reflecting what we the reality is.

If some other users tag their news with many different categories, you are right that this could blow up but I would like to leave it up to them to decide what we should do in this case :laughing:

Reply:
Ok then, I can get behind this reasoning!